thangs_fetcher.py

```diff
@@ -666,14 +481,14 @@ def get_stl_search(self, stl_path):
         self.pcoll = bpy.utils.previews.new()
         self.pcoll.Model_dir = ""
         self.pcoll.Model = ()
-        self.pcoll.ModelView1 = ()
-        self.pcoll.ModelView2 = ()
-        self.pcoll.ModelView3 = ()
-        self.pcoll.ModelView4 = ()
-        self.pcoll.ModelView5 = ()
-        self.pcoll.ModelView6 = ()
-        self.pcoll.ModelView7 = ()
-        self.pcoll.ModelView8 = ()
+        # self.pcoll.ModelView1 = ()
+        # self.pcoll.ModelView2 = ()
+        # self.pcoll.ModelView3 = ()
+        # self.pcoll.ModelView4 = ()
+        # self.pcoll.ModelView5 = ()
+        # self.pcoll.ModelView6 = ()
+        # self.pcoll.ModelView7 = ()
+        # self.pcoll.ModelView8 = ()

```

Comment:
Small nitpick, delete the commented out code.

Reply:
This is part of the clean up of get_stl_search function that @RandyHucker is doing next.